pydantic_models/essentiality.py

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+"""Generating json schema via pydantic models."""
+
+from enum import Enum
+import json
+from typing import List, Optional
+
+from pydantic import BaseModel, Extra, Field
+
+class MutationType(str, Enum):
+    """Mutation class of a target in the essentiality object."""
+    damaging = 'damaging'
+    hotspot = 'hotspot'
+
+class Screen(BaseModel):
+    depmapId: str = Field(description="Screen identifier on DepMap.", examples=['ACH-001029'])
+    cellLineName: str = Field(description="Screened cell-line name.", examples=['CHLA-10'])
+    diseaseFromSource: Optional[str] = Field(description="Disease represented by the applied cell-line.", examples=['Ewing Sarcoma'])
+    diseaseCellLineId: Optional[str] = Field(description="Disease represented by the applied cell-line.", examples=['SIDM01125'], regex=r'SIDM\d+')
+    mutation: Optional[MutationType] = Field(description='Type of the mutation class of the target.', examples=['hotspot'])
```

Comment:
I read in the issue that you only saw data for these 2 mutation types, but in Depmap they also include "Other non conserving" and "Other conserving" mutations. 
CABYR has some data points that belong to the "other" category https://depmap.org/portal/gene/CABYR?tab=dependency
Are we missing any data?

Reply:
Yes, that pieces of information is quite difficult to extract. Not for this release.